src/enumeration.rs

```diff
@@ -187,6 +189,12 @@ impl DeviceInfo {
         self.protocol
     }
 
+    /// Maximum packet size for endpoint zero.
+    #[doc(alias = "bMaxPacketSize0")]
+    pub fn max_packet_size(&self) -> u8 {
```

Comment:
I think this should be `max_packet_size_0` to rust-ify the USB spec name, since it is specific to endpoint 0.